docs/datatypes/matrices.md

```diff
@@ -181,33 +181,30 @@ Math.js uses geometric dimensions:
 - A vector is one-dimensional.
 - A matrix is two or multidimensional.
 
-The size of a matrix can be calculated with the function `size`. Function `size`
-returns a `Matrix` or `Array`, depending on the configuration option `matrix`.
-Furthermore, matrices have a function `size` as well, which always returns
-an Array.
+The size of a matrix can be calculated with the function `size`. Function size returns an `Array`, giving the length of its input (`Matrix` or `Array`) in each dimension. You can also call `size()` as a method on a Matrix.
```

Comment:
You've eliminated the code-quotes on the second occurrence of "Function `size`" making the typography potentially a little confusing. Please either say "This function" or "Function `size`".

Also, this paragraph was line-broken at 80 characters in the markdown source. Please keep that formatting in the replacement (rather than one very long line), thanks.